src/webgpu/api/validation/encoding/beginComputePass.spec.ts

```diff
@@ -106,6 +106,40 @@ g.test('timestampWrites,query_set_type')
     t.tryComputePass(isValid, descriptor);
   });
 
+g.test('timestamp_query_feature')
+  .desc(
+    `
+  Test that a validation error is generated if the features doesn't contain 'timestamp-query'
+  feature.
```

Comment:
nit: It's easier to read language which describes the intent to enable the feature, rather than the mechanism used to enable the feature.

```suggestion
  Test that a validation error if the 'timestamp-query' feature isn't enabled.
```